cube-texture-lighting.html

```diff
@@ -180,31 +180,39 @@
         usage: GPUBufferUsage.UNIFORM | GPUBufferUsage.COPY_DST
     });
 
-    device.defaultQueue.writeBuffer(vertexUniformBuffer, 64, viewProjectionMatrix);
-    device.defaultQueue.writeBuffer(fragmentUniformBuffer, 0, eyePosition);
-    device.defaultQueue.writeBuffer(fragmentUniformBuffer, 16, lightPosition);
+    device.queue.writeBuffer(vertexUniformBuffer, 64, viewProjectionMatrix);
+    device.queue.writeBuffer(fragmentUniformBuffer, 0, eyePosition);
+    device.queue.writeBuffer(fragmentUniformBuffer, 16, lightPosition);
 
     const sceneUniformBindGroupLayout = device.createBindGroupLayout({
         entries: [
             {
                 binding: 0,
                 visibility: GPUShaderStage.VERTEX,
-                type: "uniform-buffer"
+                buffer: {
+                    type: "uniform"

```

Comment:
It looks like "uniform" is the default type for buffers, so I think this could technically be `buffer: {}`. Do you have a preference between explicit vs. concise?

Reply:
Explicit, generally. Since this is meant to help people learn, I think it's better to be clear.